src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Azs.Subscriptions.Admin.psd1

```diff
@@ -17,7 +17,7 @@ Licensed under the MIT License. See License.txt in the project root for license
     RootModule        = 'Azs.Subscriptions.Admin.psm1'
 
     # Version number of this module.
-    ModuleVersion     = '0.1.0'
```

Comment:
This would be conflicting with the other PR that carl made

Reply:
If his PR does not make his in then we can use this, if not he can accept his changes.